ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -340,6 +340,39 @@ function upgrade() {
     echo "Upgrading cluster to ${TARGET} gets started..."
 }
 
+# https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-73352
+function check_upgrade_recommend_when_upgrade_inprogress() {
+    if [[ -n "${SOURCE_MINOR_VERSION}" ]] && [[ "${SOURCE_MINOR_VERSION}" -le "17" ]] ; then
```

Comment:
The test is enabled via `ENABLE_OTA_TEST` in job config yaml with explicit source and target version, so this is not necessary?

Reply:
Update the `if` block to below:
```
if [[ "${SOURCE_MINOR_VERSION}" -eq "18" ]] || [[ "${SOURCE_MINOR_VERSION}" -eq "19" ]] ; then
```